cloudify_ansible/utils.py

```diff
@@ -134,6 +138,17 @@ def _get_tenant_name(_ctx=None):
     return _ctx.tenant_name
 
 
+def _get_collections_location(_ctx=None):
+    _ctx = _ctx or ctx
+    runtime_properties = get_instance(ctx).runtime_properties
+    if not is_local_venv() and \
+            (get_node(ctx).properties.get('galaxy_collections')
+             or runtime_properties.get('galaxy_collections')):
+        return runtime_properties.get(WORKSPACE)
+    return os.path.join(runtime_properties.get(PLAYBOOK_VENV),
+                        'lib/python3.6/site-packages')
```

Comment:
@EarthmanT should we start considering [ the move to python3.10 at some point ], because if we are this will surly fail on 7.X

Reply:
that's not the ideal way of managing paths, I'll work on more dynamic approach